pkg/namesgenerator/names-generator.go

```diff
@@ -129,7 +129,7 @@ var (
 	}
 
 	// Docker, starting from 0.7.x, generates names from notable scientists and hackers.
-	// Please, for any amazing man that you add to the list, consider adding an equally amazing woman to it, and vice versa.
+	// Please, for any amazing man that you add to the list, consider adding an equally amazing person to it, and vice versa.
```

Comment:
Interesting; the bot picks out `woman` (changes it to `person`), but doesn't pick out `man` ??

Anyway, this one is on purpose (it's not perfect, as it's describing binary (man/woman), and I've thought about changing that, but given that we declared this file to be frozen, not sure if we need to change)